packages/cli/src/commands/open.ts

```diff
@@ -0,0 +1,324 @@
+import chalk from 'chalk';
+import Client from '../util/client';
+import getArgs from '../util/get-args';
+import getScope from '../util/get-scope';
+import handleError from '../util/handle-error';
+import logo from '../util/output/logo';
+import { getCommandName, getPkgName } from '../util/pkg-name';
+import validatePaths from '../util/validate-paths';
+import { ensureLink } from '../util/ensure-link';
+import list from '../util/input/list';
+import { Org, Project, Team } from '../types';
+import { stringify } from 'querystring';
+import execa from 'execa';
+import link from '../util/output/link';
+import { getDeployment } from '../util/get-deployment';
+import { normalizeURL } from '../util/normalize-url';
+
+const help = () => {
+  console.log(`
+  ${chalk.bold(`${logo} ${getPkgName()} open`)} [options]
+
+  ${chalk.dim('Options:')}
+
+    -h, --help                     Output usage information
+    --yes                          Skip confirmation prompts
+    dash                           Open the dashboard in a browser
+    latest                         Open the latest preview deployment URL in a browser
+    prod                           Open the latest production deployment URL in a browser
+    [url]                          Open the specified deployment URL in a browser
+    -A ${chalk.bold.underline('FILE')}, --local-config=${chalk.bold.underline(
+    'FILE'
+  )}   Path to the local ${'`vercel.json`'} file
+    -Q ${chalk.bold.underline('DIR')}, --global-config=${chalk.bold.underline(
+    'DIR'
+  )}    Path to the global ${'`.vercel`'} directory
+
+  ${chalk.dim('Examples:')}
+
+  ${chalk.gray('–')} View all options
+
+    ${chalk.cyan(`$ ${getPkgName()} open`)}
+
+  ${chalk.gray('–')} Open the dashboard for the current project
+
+    ${chalk.cyan(`$ ${getPkgName()} open dash`)}
+
+  ${chalk.gray('–')} Open the latest preview deployment URL
+
+    ${chalk.cyan(`$ ${getPkgName()} open latest`)}
+
+  ${chalk.gray('–')} Open the latest production deployment URL
+
+    ${chalk.cyan(`$ ${getPkgName()} open prod`)}
```

Comment:
This should probably be `vc open latest --prod` to follow the same pattern as `vc deploy` and `vc deploy --prod`.